.nf-core.yml

```diff
@@ -2,4 +2,3 @@ repository_type: pipeline
 lint:
   # We skip these linting as we have splitted tests between targeted and screening
   files_exist: conf/test.config
-  actions_ci: false
```

Comment:
why was it set to false?

Reply:
It's a linting test checking that the GitHub Action which runs the tests with `-prifile test` was there, as we modified the file, this was failing. Now it's not failing after the modifications we did with the matrix to run all test and renaming the process to `test` again.